app/models/organization/CreditTransaction.scala

```diff
@@ -0,0 +1,364 @@
+                             createdAt: Instant = Instant.now,
+                             updatedAt: Instant = Instant.now,
+                             isDeleted: Boolean = false)
+
+class CreditTransactionDAO @Inject()(organizationDAO: OrganizationDAO,
+                                     conf: WkConf,
+                                     slackNotificationService: SlackNotificationService,
+                                     sqlClient: SqlClient)(implicit ec: ExecutionContext)
+    extends SQLDAO[CreditTransaction, OrganizationCreditTransactionsRow, OrganizationCreditTransactions](sqlClient) {
+
+  protected val collection = OrganizationCreditTransactions
+
+  protected def idColumn(x: OrganizationCreditTransactions): Rep[String] = x._Id
+
+  override protected def isDeletedColumn(x: OrganizationCreditTransactions): Rep[Boolean] = x.isDeleted
+
+  override protected def parse(row: OrganizationCreditTransactionsRow): Fox[CreditTransaction] =
+    for {
+      state <- CreditTransactionState.fromString(row.state).toFox
+      id <- ObjectId.fromString(row._Id)
+      jobIdOpt <- Fox.runOptional(row._PaidJob)(ObjectId.fromStringSync)
+      refundedTransactionIdOpt <- Fox.runOptional(row.refundedTransactionId)(ObjectId.fromStringSync)
+    } yield {
+      CreditTransaction(
+        id,
+        row._Organization,
+        row.creditChange,
+        row.refundableCreditChange,
+        refundedTransactionIdOpt,
+        row.spentMoney,
+        row.comment,
+        jobIdOpt,
+        state,
+        row.expirationDate.map(Instant.fromDate),
+        Instant.fromSql(row.createdAt),
+        Instant.fromSql(row.updatedAt),
+        row.isDeleted
+      )
+    }
+
+  implicit val getOrganizationCreditTransactions: GetResult[CreditTransaction] = GetResult { prs =>
+    import prs._
+    val transactionId = <<[ObjectId]
+    val organizationId = <<[String]
+    val creditChange = <<[BigDecimal]
+    val refundableCreditChange = <<[BigDecimal]
+    val refundedTransactionId = <<?[ObjectId]
+    val spentMoney = <<?[BigDecimal]
+    val comment = <<[String]
+    val paidJobId = <<?[ObjectId]
+    val stateOpt = CreditTransactionState.fromString(<<[String])
+    val state = stateOpt.getOrElse(throw new RuntimeException(s"Unknown credit transaction state: $stateOpt"))
+    val expiresAt = <<?[Instant]
+    val createdAt = <<[Instant]
+    val updatedAt = <<[Instant]
+    val isDeleted = <<[Boolean]
+    CreditTransaction(
+      transactionId,
+      organizationId,
+      creditChange,
+      refundableCreditChange,
+      refundedTransactionId,
+      spentMoney,
+      comment,
+      paidJobId,
+      state,
+      expiresAt,
+      createdAt,
+      updatedAt,
+      isDeleted
+    )
+  }
+
+  override protected def readAccessQ(requestingUserId: ObjectId): SqlToken =
+    q"""(_id IN (SELECT _organization FROM webknossos.users_ WHERE _multiUser = (SELECT _multiUser FROM webknossos.users_ WHERE _id = $requestingUserId)))
```

Comment:
Who should be able to see the own organization’s credit balance? 
Admins, Team Managers, Members?
cc @normanrz

Reply:
Ok let’s defer this to the next iteration then.